src/BenchmarkDotNet/Environments/Runtimes/MonoRuntime.cs

```diff
@@ -15,8 +17,15 @@ public class MonoRuntime : Runtime, IEquatable<MonoRuntime>
 
         public string MonoBclPath { get; }
 
+        public bool IsDotNetBuiltIn { get; set; }
```

Comment:
I am not a big fan of this name, on the other hand I can't suggest a better name... could you please make it internal for now, so we can rename it easily in the future?

```suggestion
        internal bool IsDotNetBuiltIn { get; }
```